src/Sylius/Bundle/AdminBundle/Resources/config/grids/exchange_rate.yml

```diff
@@ -0,0 +1,39 @@
+sylius_grid:
+    grids:
+        sylius_admin_exchange_rate:
+            driver:
+                name: doctrine/orm
+                options:
+                    class: "%sylius.model.exchange_rate.class%"
+            sorting:
+                ratio: desc
+            fields:
+                ratio:
+                    type: string
+                    label: sylius.ui.ratio
+                    sortable: ~
+                baseCurrency:
+                    type: twig
+                    label: sylius.ui.base_currency
+                    path: .
+                    options:
+                        template: "@SyliusAdmin/ExchangeRate/Grid/Field/baseCurrencyName.html.twig"
+                counterCurrency:
+                    type: twig
+                    label: sylius.ui.counter_currency
+                    path: .
+                    options:
+                        template: "@SyliusAdmin/ExchangeRate/Grid/Field/counterCurrencyName.html.twig"
+            filters:
+                ratio:
+                    type: string
+                    label: sylius.ui.ratio
```

Comment:
This filter is not really useful, we can remove it 😄  We should have filter that accepts the currency and looks for all exchange rates where it is used (either "source" or "target"). I added this to backlog as a separate task with low priority.